tests/fileManifestRequestFilters.test.ts

```diff
@@ -24,36 +23,39 @@ const FILE_MANIFEST_STATE_COMPLETED: FileManifestState = {
 };
 
 const FILES_FACETS: Partial<FileFacet>[] = [
-  {
-    name: "category01",
-    termCount: 2,
-  },
-  {
-    name: "category02",
-    termCount: 3,
-  },
+  { name: "category01", termCount: 2 },
+  { name: "category02", termCount: 3 },
+  { name: "category03", termCount: 1 },
 ];
 
-const FILTER_IDENTIFIER: SelectedFilter = {
-  categoryKey: "identifier",
+const SELECTED_FILTER_ENTITY_LIST: SelectedFilter = {
+  categoryKey: "category03",
   value: ["value05"],
 };
 
-const FILTERS_COMPLETE_SET: Filters = [
+const SELECTED_FILTERS: Filters = [SELECTED_FILTER_ENTITY_LIST]; // Filters pre-selected i.e. entity list selected filters, or entity ID filter.
+
+const SELECTED_FILTERS_FORM_COMPLETE_SET: Filters = [
   { categoryKey: "category01", value: ["value01", "value02"] },
   { categoryKey: "category02", value: ["value02", "value03", "value04"] },
 ];
 
-const FILTERS_SUBSET: Filters = [
+const SELECTED_FILTERS_FORM_SUBSET: Filters = [
   { categoryKey: "category01", value: ["value01"] },
   { categoryKey: "category02", value: ["value02", "value03", "value04"] },
 ];
```

Comment:
Could you add a third test case here where one of the categories has no selected values, so that the "no values equals all values" case is exercised?